06-release-engineering-analytics.Rmd

```diff
@@ -424,11 +424,91 @@ but that is probably still part of RRs (even though it's so specific) [@castellu
 
 ### RQ3: Future Research
 
-This section will give an answer to the question:
+This section aims to answer the question:
```

Comment:
This sentence is the same for all answer subsections, shouldn't we also change this sentence in the other subsections then?

Reply:
Agreed